proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,837 @@
+
+## Sections
+
+Content sectioning elements allow you to organize the document content into logical pieces. Use the sectioning elements to create a broad outline for your page content, including header and footer navigation, and heading elements to identify sections of content.
+
+* [ ] `article` is equivalent to `View` with `display:block` layout and `role="article"`.
+* [ ] `aside`
+* [ ] `div` is equivalent to `View` with `display:block` layout.
+* [ ] `footer`.
+* [ ] `h1-6`.
+* [ ] `header`.
+* [ ] `main`.
+* [ ] `nav`.
+* [ ] `section` is equivalent to `View` with `display:block` layout and `role="section"`.
+
+## Text
+
+Text content elements organize blocks or sections of content. Important for accessibility, these elements identify the purpose or structure of that content.
+
+* [ ] `blockquote`.
+* [ ] `li`.
+* [ ] `ol`.
+* [ ] `ul`.
+* [ ] `p`.
+* [ ] `pre`.
+
+## Inline text
+
+Inline text semantics define the meaning, structure, or style of a word, line, or any arbitrary piece of text.
+
+* [ ] `a`.
+* [ ] `b`.
+* [ ] `bdi`.
+* [ ] `bdo`.
+* [ ] `code`.
+* [ ] `em`.
+* [ ] `span` is equivalent to `Text` with `display:inline` layout.
+* [ ] `strong`.
+* [ ] `sub`.
+* [ ] `sup`.
+
+## Media
+
+* [ ] `img` is equivalent to `Image` without advanced loader configuration.
+
+## Forms
+
+Elements which can be used together to create forms which the user can fill out and submit. There's a great deal of further information about this available in the [HTML forms guide](https://developer.mozilla.org/en-US/docs/Learn/Forms).
+
+* [ ] `button`.
+* [ ] `input` is equivalent to `TextInput` with `inputMode` corresponding to `type`.
+* [ ] `label`.
+* [ ] `optgroup`.
+* [ ] `option`.
+* [ ] `progress`.
+* [ ] `select`.
+* [ ] `textarea` is equivalent to `TextInput` with `multiline={true}`.
+
+-----
+
+# Elements API (DOM subset)
+
+Subset of cross-platform DOM APIs exposed on element instances, document, etc. This would be an imperative API for working with the Shadow tree in React Native. Based on data from [MDN Web API](https://developer.mozilla.org/en-US/docs/Web/API), please refer to the MDN links for complete DOM APIs and comment if you believe any should be included in the proposed subset.
+
+In practice, this DOM API subset is generally limited to the “best practice” APIs that can safely be used with React DOM (i.e., read-only operations that don’t modify the DOM tree), and most React components should already be limited to this subset.
+
+## EventTarget
```

Comment:
This API is common to make JS environments, and in the context of React Native it provides a lot of developer flexibility for building event handling and dispatching systems. A lot of React hooks built for React DOM could "just work" with the EventTarget API alone.